### DISCUSSION TITLE
Let the Brute Parser ignore capitalization

### DISCUSSION THREAD

**Discussion by RealFoxie** [mealie-recipes/mealie#1797] [Feature Request]
I wasn't sure if I should add this as a bug, or this is seen as a feature.

Currently, the brute parser does an exact match of the ingredient. Some fixes are already proposed to help with this (by adding the possibility for aliases etc). However, at this moment the Brute Parser also fails if the ingredient and input are the same, except for capitalization.
Example:
the food `onion` exists, but when parsing an ingredient with the word `Onion`, it will not work.

Would it be a good idea to lower-case every input string (and every ingredient/unit) for the parser? I can't think of an example where capitalization matters. However, I can see that keeping the capitalization of the user is needed.
One problem might be the suggestions when an ingredient does not exist. This should still have the capitalization of the input. This means that the easy solution of making the whole input lower-case is not possible. Instead it should be done at the actual test of equality. But that's implementation details :)

**Comment by michael-genson** [comment]
Pretty sure we fixed this, if you're still seeing this issue feel free to submit a bug report
